packages/demo/src/components/examples/LimitExamples.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+import {Limit} from 'react-vapor';
+
+export class LimitExamples extends React.Component {
+    render() {
```

Comment:
We tend to go with functional components instead of class components.
```tsx
export const LimitExamples: React.FunctionComponent = () => {
    // consts here
    return (
        <div>Markup here</div>
    );
};
```